workspaces/web-app/pages/index.js

```diff
@@ -1,18 +1,14 @@
 import React from 'react'
-import styled, { hydrate } from 'react-emotion'
+import styled from '@emotion/styled'
+import { css } from '@emotion/core'
 
-// Adds server generated styles to emotion cache.
-// '__NEXT_DATA__.ids' is set in '_document.js'
-/* istanbul ignore if */
-if (typeof window !== 'undefined' && window.__NEXT_DATA__) {
-  hydrate(window.__NEXT_DATA__.ids)
-}
+const headerStyles = css`
+  font-size: 20px;
+`
 
-const H1 = styled.h1(
-  {
-    fontSize: 20
-  },
-  props => ({ color: props.color })
-)
+const H1 = styled.h1`
```

Comment:
Do we really want to use interpolated strings? As you know I really prefer the glamorous syntax with object styles (https://emotion.sh/docs/styled#object-styles). I am fine with using `css` prop, but then in object notation and not with interpolated strings. One reason I am open to use `css` prop or `css` function (https://emotion.sh/docs/object-styles#with-css) more frequently instead of e.g. `styled.div({ margin: 0 })` is that by using `css` prop one can escape the situation where in every component you are creating a styled version of a `div` named `Wrapper` or `Container`. I am not convinced that it is bad per se, but some may dislike it. 

But, again, maybe you have some super-strong point about using interpolated strings... In such a case let's discuss.

Reply:
Good point